fastlane/Fastfile

```diff
@@ -32,7 +32,6 @@ platform :ios do
     match(type: "appstore")
     increment_build_number
     gym(scheme: "Tropos")
-    increment_build_number
```

Comment:
Because it's incrementing the [here also](https://github.com/thoughtbot/Tropos/pull/183#discussion-diff-66001416R33). It also doesn't do anything because at this point the archive is already built.